templates/status.html

```diff
@@ -0,0 +1,210 @@
+{{define "title"}}
+Connected Peers
+{{end}}
+
+{{define "top_css"}}
+{{end}}
+
+{{define "username"}}
+{{ .username }}
+{{end}}
+
+{{define "page_title"}}
+Connected Peers
+{{end}}
+
+{{define "page_content"}}
+<section class="content">
+    <div class="container-fluid">
+        {{ if .error }}
+        <div class="alert alert-warning" role="alert">{{.error}}</div>
+        {{ end}}
+        {{ range $dev := .devices }}
+            <table class="table table-sm">
+                <caption>List of connected peers for device with name {{ $dev.Name }} </caption>
+              <thead>
+                <tr>
+                  <th scope="col">#</th>
+                  <th scope="col">Name</th>
+                  <th scope="col">Email</th>
+                  <th scope="col">Public Key</th>
+                  <th scope="col">ReceiveBytes</th>
+                  <th scope="col">TransmitBytes</th>
+                  <th scope="col">Connected (Approximation)</th>
+                  <th scope="col">LastHandshakeTime</th>
+                </tr>
+              </thead>
+              <tbody>
+              {{ range $idx, $peer := $dev.Peers }}
+              <tr {{ if $peer.Connected }} class="table-success" {{ end }}>
+                <th scope="row">{{ $idx }}</th>
+                <td>{{ $peer.Name }}</td>
+                <td>{{ $peer.Email }}</td>
+                <td>{{ $peer.PublicKey }}</td>
+                <td>{{ $peer.ReceivedBytes }}</td>
+                <td>{{ $peer.TransmitBytes }}</td>
+                <td>{{ $peer.Connected }}</td>
+                <td>{{ $peer.LastHandshakeTime }}</td>
+               </tr>
+              {{ end }}
+              </tbody>
+            </table>
+        {{ end }}
+
+
+    </div>
+</section>
+
+<div class="modal fade" id="modal_endpoint_address_suggestion">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h4 class="modal-title">Endpoint Address Suggestion</h4>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="modal-body">
+                <p>Following is the list of public and local IP addresses for your consideration.</p>
+                <select id="ip_suggestion" class="select2"
+                    data-placeholder="Select an IP address" style="width: 100%;">
+                </select>
+            </div>
+            <div class="modal-footer justify-content-between">
+                <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+                <button type="button" class="btn btn-success" id="btn_use_ip" disabled>Use selected IP address</button>
+            </div>
+        </div>
+        <!-- /.modal-content -->
+    </div>
+    <!-- /.modal-dialog -->
+</div>
+<!-- /.modal -->
+{{end}}
+
+{{define "bottom_js"}}
```

Comment:
It looks like you cloned from the `global_settings.html` template. Do we need the following bottom_js scripts here?

Reply:
removed